xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/notification/email/macros.vm

```diff
@@ -256,7 +262,7 @@
       ## tools to display the title.
       #set ($title = $document.displayTitle)
     #end
-    <li style="vertical-align: middle; padding-left: 0; margin: 5px 0;"><span style="color: #ccc;">---</span> <img src="cid:NEPage.png" width="8" height="10" style="width: 8px; height: 10px; margin-right: 2px;"/> #link("#${escapetool.xml($ev.event.events.get(0).id)}", $title)
+    <li style="vertical-align: middle; padding-left: 0; margin: 5px 0;"><span style="color: #767676;">---</span> <img src="cid:NEPage.png" width="8" height="10" style="width: 8px; height: 10px; margin-right: 2px;"/> #link("#${escapetool.xml($ev.event.events.get(0).id)}", $title)
```

Comment:
This one is not coming from a variable?

Reply:
My bad, fixed with the other colors of the "tree" used in the table of content at the top of the mail. This tree is made with dashed borders AND `- - -` as text, it's pretty chaotic ^^`

That's the reason why there's a `borderColor` used for a `color` value :)

Fix provided in https://github.com/xwiki/xwiki-platform/pull/3037/commits/024df2b3173a742812fd7aef59d80528660c71f2 :+1: